docs/docs/api-changes.md

```diff
@@ -1,5 +1,12 @@
 # API changes
 
+## 1.3 to 1.4
+
+### API changes
+
+For security reasons, a client that sets the security level in the [`EnclaveConstraint`](api/-conclave%20-core/com.r3.conclave.common/-enclave-constraint/index.html) to insecure will no longer be able to connect to `STALE` or `SECURE` enclaves.
+The behaviour remains the same for the security levels `STALE`, and `SECURE`.
```

Comment:
Please make the following suggested changes.

1. Please use the American spelling "behavior".
2. Please use `INSECURE` in caps and with the back-ticks to render it similar to `STALE` and `SECURE`.
3. The behavior of the security levels `STALE`, and `SECURE` has not changed.